### ISSUE TITLE
Stackable discounts

### ISSUE THREAD

**Issue by sdragnev** [dlobo/org.civicrm.module.cividiscount#161]
A user should be able to receive multiple discounts if they meet the criterion.

Eg, one discount is given if the user is a current member and in addition they can apply a coupon which adds a further discount to the already discounted price.

**Comment by colemanw** [comment]
@sdragnev that would be a good feature. Are you able to contribute code or funding to make that happen?

**Comment by diegofcv** [comment]
Hi @colemanw , I'd like to know about the funding possibility on this function, as a customer would like to use this. Can you please tell me what would be that budget? Ty.

**Comment by colemanw** [comment]
Probably this should be optional, as some organizations may not wish people to "double-dip" multiple discounts.

I haven't looked at the code for this so this is a very rough guess: 5-10 hours. Core team rates are [here](https://civicrm.org/work-with-the-core-team).

**Comment by Stoob** [comment]
@colemanw agreed.  This would be a cool feature, but it should be optional because not every organization wants to stack. For example, if a 20% discount and a $20 off discount apply to the same event, it could conceivably result in a $0 registration which may not be what is desired. Some organizations specifically do NOT want to stack and instead apply the single discount with the largest benefit, if more than one discount is active.

**Comment by mlutfy** [comment]
CiviDiscount has moved to CiviCRM's Gitlab! (see #273)

If you are interested in this issue, you can follow it here:  
https://lab.civicrm.org/extensions/cividiscount/-/issues/161
